src/Authelia/Provider.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace SocialiteProviders\Authelia;
+
+use GuzzleHttp\RequestOptions;
+use InvalidArgumentException;
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
+{
+    public const IDENTIFIER = 'AUTHELIA';
+
+    protected $scopes = ['openid profile email groups'];
```

Comment:
This is an array, please make it individual keys (and set the $scopeseperator if req)